kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1039,14 +1039,14 @@ The `JUMP*` family of operations affect the current program counter.
 
     syntax UnStackOp ::= "JUMP"
  // ---------------------------
-    rule <k> JUMP DEST => #endBasicBlock... </k>
+    rule <k> JUMP DEST => #endBasicBlock ... </k>
          <pc> _ => DEST </pc>
          <jumpDests> DESTS </jumpDests>
-      requires DEST in DESTS
+      requires DESTS[DEST] ==Int 1
```

Comment:
```suggestion
      requires DEST <Int lengthBytes(DESTS) andBool DESTS[DEST] ==Int 1
```
I think we're safe from this being less than zero, but if not, we can also add `0 <=Int DEST`.